core/src/main/scala/com/yahoo/maha/core/query/QueryPipeline.scala

```diff
@@ -728,7 +728,14 @@ class DefaultQueryPipelineFactory(implicit val queryGeneratorRegistry: QueryGene
                                     requestModel: RequestModel,
                                     queryAttributes: QueryAttributes,
                                     queryGenVersion: Version): Query = {
-    val queryType = if (isOuterGroupByQuery(bestDimCandidates, bestFactCandidate, requestModel)) {
+    val isOgb = isOuterGroupByQuery(bestDimCandidates, bestFactCandidate, requestModel)
```

Comment:
Can we add a test with a request that is ogb, force engine Presto and make sure it falls back to Hive?